docs/generated/oadm_by_example_content.adoc

```diff
@@ -201,6 +201,38 @@ Manage nodes - list pods, evacuate, or mark ready
 ====
 
 
+== oadm pod-network join-projects
+Join project network
+
+====
+
+[options="nowrap"]
+----
+	# Allow project p2 to use project p1 network
```

Comment:
If you changed these manually only here then the `verify` script will complain about it. I guess it's fine to have the go-style comments, just create an issue that both the examples in openshift-sdn and the docs here need to be updated using bash comments.